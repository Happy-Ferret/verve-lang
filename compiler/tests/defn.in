(defn square (n) (mul n n))
