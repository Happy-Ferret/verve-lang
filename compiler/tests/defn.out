section STRINGS:
defn
square
n
mul
section FUNCTIONS:
fn square(1):
push_arg $0
push_arg $0
lookup $3
push $3
call
ret
